packages/request-manager/package.json

```diff
@@ -11,11 +11,12 @@
     ],
     "scripts": {
         "lint": "eslint '**/*.{ts,tsx,js}'",
-        "test:unit": "jest -c ../../jest.config.base.js",
+        "test:e2e": "jest --runInBand -c ../../jest.config.base.js",
```

Comment:
I generally agree with this approach. my idea for the future is to dissolve integration tests fixtures into respective packages. we can agree on that it will be `e2e` folder and `test:e2e` command in package.json